Source/JavaScriptCore/bytecode/SuperSampler.h

```diff
@@ -31,7 +31,7 @@ namespace JSC {
 
 class MacroAssembler;
 
-extern JS_EXPORT_PRIVATE std::atomic<uint32_t> g_superSamplerCount;
+extern "C" JS_EXPORT_PRIVATE std::atomic<uint32_t> g_superSamplerCount;
```

Comment:
I enjoy the idea of extern “C”ing a clearly C++ type. Only good things can be done with that.